### ISSUE TITLE
Verge Not Scynicng At All - Linux

### ISSUE THREAD

**Issue by shiroi-tora** [vergecurrency/verge#1030]
I am trying to sync Vergecurrency in my Linux machine but nothing happens at all. 
I downloaded latest version: Verge Leviosa version v6.0.2.0 (release build), then tried bott ./verged and ./verge-qt methods but still got the same result. 

I tried these steps and still nothing happened:
* Deleting .tor directory
* Deleting .VERGE directory
* Creating a VERGE.conf file and adding nodes manually
* Running ./verged and ./verge-qt with --without-tor
* Running ./verged and ./verge-qt with -listenonion=0
* Running ./verged and ./verge-qt with -dynamic-network because verge said `[DEPRECATED] flag -without-tor will be removed with the upcoming release`
* Installed verge blockchain from [https://verge-blockchain.com/down](url) and did exactly what [https://verge-blockchain.com/howto](url) said. After this particular step, installed blocks did seen in my verge GUI but still sync didn't start at all. 

My full log is as follow:
`beyazkaplan@ubuntu:~/Desktop$ ./verged --without-tor`
`2020-03-23T11:04:39Z Verge Leviosa version v6.0.2.0 (release build)`
`2020-03-23T11:04:39Z InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1`
`2020-03-23T11:04:39Z Assuming ancestors of block c766387a2e0cd6af995ea432518614824fe313e988598ea8b26f58efb99ebcdc have valid signatures.`
`2020-03-23T11:04:39Z Setting nMinimumChainWork=000000000000000000000000000000000000000000000000000000222e000001`
`2020-03-23T11:04:39Z ParameterInteraction: Using algo: scrypt 0`
`2020-03-23T11:04:39Z Using the 'sse4(1way+4way),avx2(8way)' SHA256 implementation`
`2020-03-23T11:04:39Z Using RdRand as an additional entropy source`
`2020-03-23T11:04:39Z Default data directory /home/beyazkaplan/.VERGE`
`2020-03-23T11:04:39Z Using data directory /home/beyazkaplan/.VERGE`
`2020-03-23T11:04:39Z Using config file /home/beyazkaplan/.VERGE/VERGE.conf`
`2020-03-23T11:04:39Z Using at most 125 automatic connections (1024 file descriptors available)`
`2020-03-23T11:04:39Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements`
`2020-03-23T11:04:39Z Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements`
`2020-03-23T11:04:39Z Using 8 threads for script verification`
`2020-03-23T11:04:39Z scheduler thread start`
`2020-03-23T11:04:39Z HTTP: creating work queue of depth 16`
`2020-03-23T11:04:39Z No rpcpassword set - using random cookie authentication`
`2020-03-23T11:04:39Z Generated RPC authentication cookie /home/beyazkaplan/.VERGE/.cookie`
`2020-03-23T11:04:39Z HTTP: starting 4 worker threads`
`2020-03-23T11:04:39Z Using wallet directory /home/beyazkaplan/.VERGE/wallets`
`2020-03-23T11:04:39Z init message: Verifying wallet(s)...`
`2020-03-23T11:04:39Z Using BerkeleyDB version Berkeley DB 4.8.30: (April  9, 2010)`
`2020-03-23T11:04:39Z Using wallet wallet.dat`
`2020-03-23T11:04:39Z BerkeleyEnvironment::Open: LogDir=/home/beyazkaplan/.VERGE/wallets/database ErrorFile=/home/beyazkaplan/.VERGE/wallets/db.log`
`2020-03-23T11:04:39Z [DEPRECATED] flag -without-tor will be removed with the upcoming release`
`2020-03-23T11:04:39Z [DEPRECATED] Please use -dynamic-network instead`
`2020-03-23T11:04:39Z [DEPRECATED] Tor disabled, Socks Proxy not initialized.`
`2020-03-23T11:04:39Z Cache configuration:`
`2020-03-23T11:04:39Z * Using 2.0MiB for block index database`
`2020-03-23T11:04:39Z * Using 8.0MiB for chain state database`
`2020-03-23T11:04:39Z * Using 440.0MiB for in-memory UTXO set (plus up to 286.1MiB of unused mempool space)`
`2020-03-23T11:04:39Z init message: Loading block index...`
`2020-03-23T11:04:39Z Opening LevelDB in /home/beyazkaplan/.VERGE/blocks/index`
`2020-03-23T11:04:39Z Opened LevelDB successfully`
`2020-03-23T11:04:39Z Using obfuscation key for /home/beyazkaplan/.VERGE/blocks/index: 0000000000000000`
`2020-03-23T11:04:53Z LoadBlockIndexDB: last block file = 41`
`2020-03-23T11:04:53Z LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=43004, size=36468529, heights=3826579...3869348, time=2020-02-13...2020-03-01)`
`2020-03-23T11:04:53Z Checking all blk files are present...`
`2020-03-23T11:04:53Z Opening LevelDB in /home/beyazkaplan/.VERGE/chainstate`
`2020-03-23T11:04:53Z Opened LevelDB successfully`
`2020-03-23T11:04:53Z Using obfuscation key for /home/beyazkaplan/.VERGE/chainstate: 7f37befa5a409a64`
`2020-03-23T11:04:54Z Loaded best chain: hashBestChain=0000000000013ea1124176b11a81e752d8dd1c55a46eaf334f54dcb027a4e989 height=3869348 date=2020-03-01T09:50:52Z progress=0.990820`
`2020-03-23T11:04:54Z init message: Rewinding blocks...`
`2020-03-23T11:05:09Z init message: Verifying blocks...`
`2020-03-23T11:05:09Z Verifying last 6 blocks at level 3`
`2020-03-23T11:05:09Z [0%]...[16%]...[33%]...[50%]...[66%]...[83%]...[99%]...[DONE].`
`2020-03-23T11:05:09Z No coin database inconsistencies in last 6 blocks (10 transactions)`
`2020-03-23T11:05:09Z  block index           30005ms`
`2020-03-23T11:05:09Z init message: Loading wallet...`
`2020-03-23T11:05:09Z nFileVersion = 6000200`
`2020-03-23T11:05:09Z Keys: 2001 plaintext, 0 encrypted, 2001 w/ metadata, 2001 total. Unknown wallet records: 1`
`2020-03-23T11:05:09Z  wallet                   16ms`
`2020-03-23T11:05:09Z setKeyPool.size() = 2000`
`2020-03-23T11:05:09Z mapWallet.size() = 0`
`2020-03-23T11:05:09Z mapAddressBook.size() = 0`
`2020-03-23T11:05:09Z mapBlockIndex.size() = 3873663`
`2020-03-23T11:05:09Z nBestHeight = 3869348`
`2020-03-23T11:05:09Z Bound to [::]:21102`
`2020-03-23T11:05:09Z Bound to 0.0.0.0:21102`
`2020-03-23T11:05:09Z init message: Loading P2P addresses...`
`2020-03-23T11:05:09Z Loaded 3 addresses from peers.dat  0ms`
`2020-03-23T11:05:09Z init message: Loading banlist...`
`2020-03-23T11:05:09Z init message: Starting network threads...`
`2020-03-23T11:05:09Z net thread start`
`2020-03-23T11:05:09Z dnsseed thread start`
`2020-03-23T11:05:09Z addcon thread start`
`2020-03-23T11:05:09Z init message: Done loading`
`2020-03-23T11:05:09Z opencon thread start`
`2020-03-23T11:05:09Z msghand thread start`
`2020-03-23T11:05:09Z Imported mempool transactions from disk: 0 succeeded, 0 failed, 0 expired, 0 already there`
`2020-03-23T11:05:20Z Loading addresses from DNS seeds (could take a while)`
`2020-03-23T11:05:21Z One Shot the seed: seed.marpmedev.xyz`
`2020-03-23T11:05:23Z One Shot the seed: seed.verge.dev`
`2020-03-23T11:05:23Z One Shot the seed: 185.162.9.97`
`2020-03-23T11:05:25Z One Shot the seed: 159.89.46.252`
`2020-03-23T11:05:27Z One Shot the seed: 139.59.34.170`
`2020-03-23T11:05:28Z One Shot the seed: 134.209.197.243`
`2020-03-23T11:05:28Z One Shot the seed: 104.131.144.82`
`2020-03-23T11:05:28Z One Shot the seed: 192.241.187.222`
`2020-03-23T11:05:30Z One Shot the seed: 105.228.198.44`
`2020-03-23T11:05:31Z One Shot the seed: 46.127.57.167`
`2020-03-23T11:05:32Z One Shot the seed: 98.5.123.15`
`2020-03-23T11:05:34Z One Shot the seed: 81.147.68.236`
`2020-03-23T11:05:36Z New outbound peer connected: version: 90008, blocks=3924433, peer=0`
`2020-03-23T11:05:36Z One Shot the seed: 77.67.46.100`
`2020-03-23T11:05:37Z One Shot the seed: 95.46.99.96`
`2020-03-23T11:05:39Z One Shot the seed: 138.201.91.159`
`2020-03-23T11:05:41Z One Shot the seed: 159.89.202.56`
`2020-03-23T11:05:41Z One Shot the seed: 163.158.20.118`
`2020-03-23T11:05:42Z New outbound peer connected: version: 90008, blocks=3924434, peer=1`
`2020-03-23T11:05:42Z One Shot the seed: 99.45.88.147`
`2020-03-23T11:05:42Z New outbound peer connected: version: 90008, blocks=3924434, peer=2`
`2020-03-23T11:05:43Z One Shot the seed: 114.145.237.35`
`2020-03-23T11:05:44Z One Shot the seed: 73.247.117.99`
`2020-03-23T11:05:46Z One Shot the seed: 145.239.0.122`
`2020-03-23T11:05:46Z 0 addresses found from DNS seeds`
`2020-03-23T11:05:46Z dnsseed thread exit`
`2020-03-23T11:15:54Z Potential stale tip detected, will try using extra outbound peer (last tip update: 630 seconds ago) `
`2020-03-23T11:26:24Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1260 seconds ago)`
`2020-03-23T11:36:54Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1890 seconds ago)`
`2020-03-23T11:47:24Z Potential stale tip detected, will try using extra outbound peer (last tip update: 2520 seconds ago)`

* Verge Release: v6.0.2.0
* OS: I'm using Linux

What should I do now?

**Comment by justinvforvendetta** [comment]
ok a few questions. are you running other coin wallets? have you tried closing the wallet/daemon, and deleting the peers.dat, and opening it again?

**Comment by shiroi-tora** [comment]
I am not using any other coin wallets. I tried closing wallet, deleted peers.dat but unfortunately result seems to be same. 

An extra information, I don't think it would be a problem but I will say it anyway; my linux machine is working as virtual machine on VMware Workstation. But I was able to sync Bitcoin, Zcash, Dash and Monero...

**Comment by justinvforvendetta** [comment]
ok try closing the wallet. delete the peers.dat (again) and then start it with `--without-tor`

**Comment by shiroi-tora** [comment]
Nothing changed. I tried running as admin by `sudo` command but result is same.

**Comment by justinvforvendetta** [comment]
try shutting down, deleting the blockchain data that you downloaded, and syncing from 0 yourself.

**Comment by shiroi-tora** [comment]
Deleted .VERGE, deleted .tor, deleted verged and verge-qt, redownloaded verged and verge-qt from [https://github.com/vergecurrency/VERGE/releases](url), retried running `sudo ./verge-qt --without-tor` but result is still same unfortunately.

**Comment by justinvforvendetta** [comment]
try ./verge-qt -dynamic-network

**Comment by shiroi-tora** [comment]
Nothing changed unfortunately.

**Comment by justinvforvendetta** [comment]
can  you paste the contents of your debug.log again

**Comment by shiroi-tora** [comment]
Here you are:

`2020-03-23T17:22:13Z 




2020-03-23T17:22:13Z Verge Leviosa version v6.0.2.0 (release build)
2020-03-23T17:22:13Z InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1
2020-03-23T17:22:13Z Assuming ancestors of block c766387a2e0cd6af995ea432518614824fe313e988598ea8b26f58efb99ebcdc have valid signatures.
2020-03-23T17:22:13Z Setting nMinimumChainWork=000000000000000000000000000000000000000000000000000000222e000001
2020-03-23T17:22:13Z ParameterInteraction: Using algo: scrypt 0
2020-03-23T17:22:13Z Using the 'sse4(1way+4way),avx2(8way)' SHA256 implementation
2020-03-23T17:22:13Z Using RdRand as an additional entropy source
2020-03-23T17:22:13Z Default data directory /home/beyazkaplan/.VERGE
2020-03-23T17:22:13Z Using data directory /home/beyazkaplan/.VERGE
2020-03-23T17:22:13Z Using config file /home/beyazkaplan/.VERGE/VERGE.conf
2020-03-23T17:22:13Z Using at most 125 automatic connections (1024 file descriptors available)
2020-03-23T17:22:13Z Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements
2020-03-23T17:22:13Z Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements
2020-03-23T17:22:13Z Using 8 threads for script verification
2020-03-23T17:22:13Z scheduler thread start
2020-03-23T17:22:13Z Using wallet directory /home/beyazkaplan/.VERGE/wallets
2020-03-23T17:22:13Z init message: Verifying wallet(s)...
2020-03-23T17:22:13Z Using BerkeleyDB version Berkeley DB 4.8.30: (April  9, 2010)
2020-03-23T17:22:13Z Using wallet wallet.dat
2020-03-23T17:22:13Z BerkeleyEnvironment::Open: LogDir=/home/beyazkaplan/.VERGE/wallets/database ErrorFile=/home/beyazkaplan/.VERGE/wallets/db.log
2020-03-23T17:22:13Z Tor Successfully bound ...
2020-03-23T17:22:13Z torcontroller thread start
2020-03-23T17:22:13Z Cache configuration:
2020-03-23T17:22:13Z * Using 2.0MiB for block index database
2020-03-23T17:22:13Z * Using 8.0MiB for chain state database
2020-03-23T17:22:13Z * Using 440.0MiB for in-memory UTXO set (plus up to 286.1MiB of unused mempool space)
2020-03-23T17:22:13Z init message: Loading block index...
2020-03-23T17:22:13Z Opening LevelDB in /home/beyazkaplan/.VERGE/blocks/index
2020-03-23T17:22:13Z Opened LevelDB successfully
2020-03-23T17:22:13Z Using obfuscation key for /home/beyazkaplan/.VERGE/blocks/index: 0000000000000000
2020-03-23T17:22:13Z LoadBlockIndexDB: last block file = 0
2020-03-23T17:22:13Z LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=1, size=181, heights=0...0, time=2014-10-09...2014-10-09)
2020-03-23T17:22:13Z Checking all blk files are present...
2020-03-23T17:22:13Z Opening LevelDB in /home/beyazkaplan/.VERGE/chainstate
2020-03-23T17:22:13Z Opened LevelDB successfully
2020-03-23T17:22:13Z Using obfuscation key for /home/beyazkaplan/.VERGE/chainstate: d2dd991a9a184508
2020-03-23T17:22:13Z Loaded best chain: hashBestChain=00000fc63692467faeb20cdb3b53200dc601d75bdfa1001463304cc790d77278 height=0 date=2014-10-09T18:22:44Z progress=0.000000
2020-03-23T17:22:13Z init message: Rewinding blocks...
2020-03-23T17:22:13Z init message: Verifying blocks...
2020-03-23T17:22:13Z  block index               6ms
2020-03-23T17:22:13Z init message: Loading wallet...
2020-03-23T17:22:13Z nFileVersion = 6000200
2020-03-23T17:22:13Z Keys: 2001 plaintext, 0 encrypted, 2001 w/ metadata, 2001 total. Unknown wallet records: 1
2020-03-23T17:22:13Z  wallet                   18ms
2020-03-23T17:22:13Z setKeyPool.size() = 2000
2020-03-23T17:22:13Z mapWallet.size() = 0
2020-03-23T17:22:13Z mapAddressBook.size() = 0
2020-03-23T17:22:13Z mapBlockIndex.size() = 1
2020-03-23T17:22:13Z nBestHeight = 0
2020-03-23T17:22:13Z Imported mempool transactions from disk: 0 succeeded, 0 failed, 0 expired, 0 already there
2020-03-23T17:22:13Z torcontrol thread start
2020-03-23T17:22:13Z Bound to [::]:21102
2020-03-23T17:22:13Z Bound to 0.0.0.0:21102
2020-03-23T17:22:13Z init message: Loading P2P addresses...
2020-03-23T17:22:13Z Loaded 3 addresses from peers.dat  0ms
2020-03-23T17:22:13Z init message: Loading banlist...
2020-03-23T17:22:13Z init message: Starting network threads...
2020-03-23T17:22:13Z net thread start
2020-03-23T17:22:13Z dnsseed thread start
2020-03-23T17:22:13Z addcon thread start
2020-03-23T17:22:13Z opencon thread start
2020-03-23T17:22:13Z init message: Done loading
2020-03-23T17:22:13Z msghand thread start
2020-03-23T17:22:13Z GUI: Platform customization: "other"
2020-03-23T17:22:13Z tor: Got service ID puytjw2me7qi2wuin74isit6252jfu36l2rjh2nbqagtp6d64oaqhhid, advertising service puytjw2me7qi2wuin74isit6252jfu36l2rjh2nbqagtp6d64oaqhhid.onion:21102
2020-03-23T17:22:13Z AddLocal(puytjw2me7qi2wuin74isit6252jfu36l2rjh2nbqagtp6d64oaqhhid.onion:21102,4)
2020-03-23T17:22:13Z GUI: PaymentServer::LoadRootCAs: Loaded  132  root certificates
2020-03-23T17:22:24Z Loading addresses from DNS seeds (could take a while)
2020-03-23T17:22:26Z One Shot the seed: aizckfmksnk54gqcg5fur22cz22vzut2ig7cexz45pfm7witswzawcqd.onion
2020-03-23T17:22:26Z One Shot the seed: zca2vot3ucw6bjvdg6yhgjxzvyqv7fl27pvu4pskdng6w6dhjfrs5myd.onion
2020-03-23T17:22:28Z One Shot the seed: ho6iij2ag6fcnwnqhfwkzfvitcvbrr3lmlgetxlve2isvfqp72vwu5ad.onion
2020-03-23T17:22:29Z One Shot the seed: obtsnd4wobjuubco4v2awt534zoog2wtq3oknloibmeul3dh5cxycjqd.onion
2020-03-23T17:22:30Z One Shot the seed: jdaootwqo25qcfq3qjcantgfwlilzzc72e6ormt6t3gnafeacjzgd3qd.onion
2020-03-23T17:22:30Z One Shot the seed: ry7yyqv4oeu2kjygco5xvcvi2sgr5bxkynvh6yctftghwe2jy6w2wkqd.onion
2020-03-23T17:22:31Z One Shot the seed: f3cy2jkmfl3qzqpk4srruhpefkunq3rj6wi7z57yzinyxpk3rmbluoad.onion
2020-03-23T17:22:33Z One Shot the seed: 25hj4mdmslzvu2lngo4jcsbrouwsdktv6ps3xifipju74oqyoply6qad.onion
2020-03-23T17:22:34Z One Shot the seed: amafdqgkmtbkld45kaal5cwwfbrsgnimw77gawwltsaklgxxlubjvhid.onion
2020-03-23T17:22:35Z One Shot the seed: n7rk4xqurrvedhhghkkvz2pmxalgmoviokgjiwjgcvpcxa6piym5m2ad.onion
2020-03-23T17:22:36Z One Shot the seed: rje6q245yhiyn4setn5abjlcqwapxzgwbfksrscexyhv7ffjdasmvaqd.onion
2020-03-23T17:22:39Z One Shot the seed: wu6bebmvs6jw2t6lmjg3aako5synunxm4i3zb7bybvzrvxe277achfad.onion
2020-03-23T17:22:41Z One Shot the seed: jvnd4gjzhmuiaph5xft6ju4twt5gjmpbqdc2sis7gd2pnnqpwwpqz5id.onion
2020-03-23T17:22:42Z One Shot the seed: wuoki3rq3htqmtrmrtkqtnxzgfsgclwj7jrxk35abkwksni2asdepzyd.onion
2020-03-23T17:22:43Z One Shot the seed: 3czrw4cxofudpyhl46whvcyzgmkciaai2uw7s6motxejb3fijdkgmuqd.onion
2020-03-23T17:22:44Z One Shot the seed: 7wg3vv67i5m3uiectqw4taj5pm3qmxnlgyb7ks7e5oilo62g7mxn2zqd.onion
2020-03-23T17:22:45Z One Shot the seed: y2y2cr6h4dn3g5l4545mj7dw245i3fsuixx6he6r4w4aqh3j23pwqoid.onion
2020-03-23T17:22:45Z One Shot the seed: hrpy4r4xijwps32lknzbyd36ad6vaibumceioaxe62smpiv22oktiwad.onion
2020-03-23T17:22:45Z One Shot the seed: oy2q3dl6pupe4mk2dzrbtbvpf3fjru2ajdr4rsrxlxpaweqn3utvxzad.onion
2020-03-23T17:22:46Z One Shot the seed: u7ikxx5hloxuvyr53mwqdzgxr5cjpabqruzqrx5qfcvuv6kpa22gsjid.onion
2020-03-23T17:22:47Z One Shot the seed: jno3zpfsdgrtdaxlyuowipokbumoidyecmczrjru7tjfovfjkztl2pad.onion
2020-03-23T17:22:48Z One Shot the seed: 763co2copdnav2ik2jlq33wzj2rogt4wfexjvukx5tg3tiepm2ahbrid.onion
2020-03-23T17:22:49Z One Shot the seed: yvvioyzj3w5k6z64urv55xuh65oftcy5wts4tgwpi6hxdeixamsn63qd.onion
2020-03-23T17:22:49Z One Shot the seed: zwvqhwne3mlefxd52q35cckzdo46uhvcuvl7qdzqlatgpfqe4r43suid.onion
2020-03-23T17:22:50Z One Shot the seed: ojmwnkopgmpotm2byx7vsm7xplqxhctutw3d3ve7k5mbybsuydfk46qd.onion
2020-03-23T17:22:50Z One Shot the seed: g7cbcrcxdy7kszjmvw62bfm5douesjtkioqycc6m4gpomybcsohba7ad.onion
2020-03-23T17:22:51Z One Shot the seed: 2zju2kcvt55h3v5dnghtduw5hrqqlt6nggngtyjnrjgybu7tqz4wvdad.onion
2020-03-23T17:22:52Z One Shot the seed: 3zal3m6ixo3vhvaeihybqi4r7vs33l2a4x7qce2gb6seqog2sk67lkyd.onion
2020-03-23T17:22:53Z One Shot the seed: jcq2di2z5t2x6xibjuak4syslkx46rkid6vpuzys3rnn65vhwfywhrid.onion
2020-03-23T17:22:54Z One Shot the seed: seed.marpmedev.xyz
2020-03-23T17:22:55Z One Shot the seed: seed.verge.dev
2020-03-23T17:22:55Z One Shot the seed: 185.162.9.97
2020-03-23T17:22:55Z One Shot the seed: 159.89.46.252
2020-03-23T17:22:56Z One Shot the seed: 139.59.34.170
2020-03-23T17:22:57Z One Shot the seed: 134.209.197.243
2020-03-23T17:22:57Z One Shot the seed: 104.131.144.82
2020-03-23T17:22:59Z One Shot the seed: 192.241.187.222
2020-03-23T17:23:01Z One Shot the seed: 105.228.198.44
2020-03-23T17:23:03Z One Shot the seed: 46.127.57.167
2020-03-23T17:23:05Z One Shot the seed: 98.5.123.15
2020-03-23T17:23:05Z One Shot the seed: 81.147.68.236
2020-03-23T17:23:06Z One Shot the seed: 77.67.46.100
2020-03-23T17:23:06Z One Shot the seed: 95.46.99.96
2020-03-23T17:23:08Z One Shot the seed: 138.201.91.159
2020-03-23T17:23:09Z One Shot the seed: 159.89.202.56
2020-03-23T17:23:09Z One Shot the seed: 163.158.20.118
2020-03-23T17:23:11Z One Shot the seed: 99.45.88.147
2020-03-23T17:23:12Z One Shot the seed: 114.145.237.35
2020-03-23T17:23:13Z One Shot the seed: 73.247.117.99
2020-03-23T17:23:13Z One Shot the seed: 145.239.0.122
2020-03-23T17:23:13Z 0 addresses found from DNS seeds
2020-03-23T17:23:13Z dnsseed thread exit`

**Comment by justinvforvendetta** [comment]
hmm i am connecting to some of those seeds. what is the result of `netstat | grep 21102`

**Comment by shiroi-tora** [comment]
Nothing appeared.

**Comment by justinvforvendetta** [comment]
try leaving it open for a while then. you should get a connection. i dont see anything wrong in your debug file. you sure you can access internet right now on that vm? i know u said you did before, but now also?

**Comment by shiroi-tora** [comment]
I can access internet, yes. I also read leaving it open a while would solve the problem but nothing was happened. Will try letting it open this night also.

**Comment by justinvforvendetta** [comment]
anything new at the bottom of your debug.log file? i have checked those nodes and they are working, i deleted my peers.dat and it connected right away. you do not have tor running right?

**Comment by shiroi-tora** [comment]
Here you are:

````2020-03-23T17:33:19Z New outbound peer connected: version: 90008, blocks=3925103, peer=0
````2020-03-23T17:33:21Z New outbound peer connected: version: 90008, blocks=3925103, peer=1
````2020-03-23T17:33:21Z New outbound peer connected: version: 90008, blocks=3925103, peer=2
````2020-03-23T17:33:28Z Potential stale tip detected, will try using extra outbound peer (last tip update: 630 seconds ago)
````2020-03-23T17:43:58Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1260 seconds ago)
````2020-03-23T17:54:28Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1890 seconds ago)
````2020-03-23T18:04:58Z Potential stale tip detected, will try using extra outbound peer (last tip update: 2520 seconds ago)

**Comment by justinvforvendetta** [comment]
hmm. yes leave it up, it is trying peers still. not sure why it is finding a stale tip, is the time/date on your vm accurate? i know often vm's clocks will pause when the machine isnt on. make sure the clock is up to date.

**Comment by justinvforvendetta** [comment]
actually judging by the time of the timestamps in the debug, and your posts, i think your vm clock is off.

**Comment by shiroi-tora** [comment]
I sycnronized guest time with host and retried, again deleted everyting and so but still same outcome.

**Comment by shiroi-tora** [comment]
I just got new error/warning as:

`2020-03-23T19:17:37Z 0 addresses found from DNS seeds`
`2020-03-23T19:17:37Z dnsseed thread exit`
`2020-03-23T19:18:30Z Adding fixed seed nodes as DNS doesn't seem to be available.`

**Comment by justinvforvendetta** [comment]
when you type `date` it returns the correct time now?

**Comment by justinvforvendetta** [comment]
let it stay open for a bit, and see what happens. i cant reproduce this..

**Comment by shiroi-tora** [comment]
It said now:

`2020-03-23T19:28:43Z Potential stale tip detected, will try using extra outbound peer (last tip update: 675 seconds ago)`
`2020-03-23T19:29:01Z New outbound peer connected: version: 90008, blocks=3925259, peer=1`
`2020-03-23T19:29:03Z New outbound peer connected: version: 90008, blocks=3925259, peer=2`
`2020-03-23T19:29:09Z New outbound peer connected: version: 90008, blocks=3925259, peer=3`

Is it good news? Nothing seems to be changed.

**Comment by justinvforvendetta** [comment]
are you running a qt or daemon?

**Comment by shiroi-tora** [comment]
qt

**Comment by justinvforvendetta** [comment]
ok and at the bottom right, hover mouse over network icon, 0 connections?

**Comment by shiroi-tora** [comment]
Yes, 

> 0 active connections to VERGE networks.

**Comment by justinvforvendetta** [comment]
now that you have left it open for an hour, can you paste anything new from debug.log?

**Comment by shiroi-tora** [comment]
Here you are:

````2020-03-23T19:17:37Z 0 addresses found from DNS seeds
````2020-03-23T19:17:37Z dnsseed thread exit
````2020-03-23T19:18:30Z Adding fixed seed nodes as DNS doesn't seem to be available.
````2020-03-23T19:28:43Z Potential stale tip detected, will try using extra outbound peer (last tip update: 675 seconds ago)
````2020-03-23T19:29:01Z New outbound peer connected: version: 90008, blocks=3925259, peer=1
````2020-03-23T19:29:03Z New outbound peer connected: version: 90008, blocks=3925259, peer=2
````2020-03-23T19:29:09Z New outbound peer connected: version: 90008, blocks=3925259, peer=3
````2020-03-23T19:39:13Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1305 seconds ago)
````2020-03-23T19:49:43Z Potential stale tip detected, will try using extra outbound peer (last tip update: 1935 seconds ago)
````2020-03-23T20:00:13Z Potential stale tip detected, will try using extra outbound peer (last tip update: 2565 seconds ago)
````2020-03-23T20:10:44Z Potential stale tip detected, will try using extra outbound peer (last tip update: 3196 seconds ago)
````2020-03-23T20:21:14Z Potential stale tip detected, will try using extra outbound peer (last tip update: 3826 seconds ago)
````2020-03-23T20:31:44Z Potential stale tip detected, will try using extra outbound peer (last tip update: 4456 seconds ago)
````2020-03-23T20:42:14Z Potential stale tip detected, will try using extra outbound peer (last tip update: 5086 seconds ago)
````2020-03-23T20:52:44Z Potential stale tip detected, will try using extra outbound peer (last tip update: 5716 seconds ago)
````2020-03-23T21:03:14Z Potential stale tip detected, will try using extra outbound peer (last tip update: 6346 seconds ago)
````2020-03-23T21:13:45Z Potential stale tip detected, will try using extra outbound peer (last tip update: 6977 seconds ago)
````2020-03-23T21:24:15Z Potential stale tip detected, will try using extra outbound peer (last tip update: 7607 seconds ago)

**Comment by shiroi-tora** [comment]
By the way, my linux machine date was `2020-04-24T00:24:15Z` but on the debug file it was `2020-03-23T21:24:15Z`
Could this be the thing causing problem?

**Comment by justinvforvendetta** [comment]
yes definitely;//

**Comment by shiroi-tora** [comment]
How can I solve this? I have no idea.

**Comment by justinvforvendetta** [comment]
https://www.tecmint.com/synchronize-time-with-ntp-in-linux/

**Comment by justinvforvendetta** [comment]
@shiroi-tora has this been resolved?

**Comment by shiroi-tora** [comment]
I tried solutions mentioned above but nothing changed. 

Also, I deleted my current virtual linux machine, created a new, installed verge-qt, run `sudo ./verged --dynamic-network` but still got the error. 

I run `date` command in new machine and it gave me true and synched date with my windows machine. 

I think the problem is, `./verged` command doesn't care about local UTC date as mine is UTC+3 but verge insist on just UTC. 

I don't know how to force  `./verged` to use UTC+3.
